packages/ai-jsx/src/stream/index.ts

```diff
@@ -94,6 +94,7 @@ export function toStreamResponse(renderable: Renderable): Response {
       status: 200,
       headers: {
         'Content-Type': 'text/event-stream',
+        'x-powered-by': 'ai-jsx',
```

Comment:
This feels more like a vanity header than something that gets interpreted by the client. I was thinking of setting `Content-Type` to something like `text/event-stream-ai-jsx`, but that seems bad because this is a valid SSE stream, so I want to leave this header as-is.

Any other suggestions for what header to set?